Styles/styles.css

```diff
@@ -0,0 +1,111 @@
+h1 {
+    position: absolute;
```

Comment:
Don't use `position: absolute;` everywhere. Use it if you want to overlay some element or have other reasons to remove it from the normal document flow.

Reply:
Read [this](https://developer.mozilla.org/en-US/docs/Web/CSS/position)